### PR TITLE
Update accessibility statement

### PR DESCRIPTION
**Pull request by gemmadallmandfe**
### Trello card
https://trello.com/c/D2InRSsx

### Context
We need to update our accessibility statement following the DAC accessibility audit and associated changes.

### CODE REVIEW

**Comment by gemmadallmandfe** [comment]
@GayeCord Let me know if you are happy with this:
https://review-get-into-teaching-app-3029.london.cloudapps.digital/accessibility

File: app/views/content/accessibility.md [hunk 1]
```diff
@@ -26,60 +17,63 @@ keywords:
...
 
-We'll consider your request and get back to you within 5 working days.
-Reporting accessibility problems with this service
+We’ll consider your request and get back to you within 5 days.
 
-We're always looking to improve the accessibility of this service.
+## Reporting accessibility problems with this website
 
-If you find any problems that are not listed on this page or think we're not meeting accessibility requirements email getintoteaching.helpdesk@education.gov.uk
+We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, contact our helpdesk by emailing getintoteaching.helpdesk@education.gov.uk.
```
Comment (ethax-ross) [inline app/views/content/accessibility.md:50]: Does this email address want to be a mailto hyperlink?

**Review by ethax-ross — APPROVED** [review]

File: app/views/content/accessibility.md [hunk 2]
```diff
@@ -26,60 +17,63 @@ keywords:
...
-## What to do if you cannot access parts of this service
+## Feedback and contact information
 
-[Email us](mailto:getintoteaching.helpdesk@education.gov.uk) if you need information on this service in any of the following formats:
+If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille:
 
-* accessible PDF
-* audio recording
-* braille, easy read or large print format email
+* email getintoteaching.helpdesk@education.gov.uk
```
Comment (gemmadallmandfe) [inline app/views/content/accessibility.md:44]:
```suggestion
* email [getintoteaching.helpdesk@education.gov.uk](mailto:getintoteaching.helpdesk@education.gov.uk)
```

File: app/views/content/accessibility.md (hunk 1 above)
Comment (gemmadallmandfe) [inline app/views/content/accessibility.md:50]:
```suggestion
We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, contact our helpdesk by emailing [getintoteaching.helpdesk@education.gov.uk](mailto:getintoteaching.helpdesk@education.gov.uk).
```

**Comment by GayeCord** [comment]
@gemmadallmandfe i'm happy :-)

**Pull request merged**